examples/vnet/kubernetesvnet.json.pretest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+az network vnet create -g ${RESOURCE_GROUP} -n KubernetesCustomVNET --address-prefix 10.239.0.0/16 --subnet-name KubernetesSubnet --subnet-prefix 10.239.0.0/16
```

Comment:
Does the script need to be duplicated? Seems like it's the same exact the name of the orchestrator. Can it be removed or parameterized?

This is a nit, if you think this copy/paste is okay, then I'm okay with it.